modules/video/include/opencv2/video/tracking.hpp

```diff
@@ -853,9 +853,9 @@ class CV_EXPORTS_W TrackerDaSiamRPN : public Tracker
  *
  *  Nano tracker is much faster and extremely lightweight due to special model structure, the whole model size is about 1.9 MB.
  *  Nano tracker needs two models: one for feature extraction (backbone) and the another for localization (neckhead).
- *  Please download these two onnx models at:https://github.com/HonglinChu/SiamTrackers/tree/master/NanoTrack/models/nanotrackv2.
- *  Original repo is here: https://github.com/HonglinChu/NanoTrack
- *  Author:HongLinChu, 1628464345@qq.com
+ *  Please download these two onnx models at: https://github.com/HonglinChu/SiamTrackers/tree/master/NanoTrack/models/nanotrackv2.
+ *  Original repo is here: https://github.com/HonglinChu/NanoTrack.
```

Comment:
Dots are part of URL?

Reply:
Thanks for your reviewing, removed the final Dots.